scripts/fix-ca-for-k3s.sh

```diff
@@ -0,0 +1,46 @@
+#! /bin/sh
```

Comment:
Let's use consistent style for scripts in federation-v2. `s,#! /bin/sh,#!/usr/bin/env bash,`

Reply:
i'm not sure if i agree it's a style matter.
but anyway i made it similar to others, including pipefail thing.